### ISSUE TITLE
Exporting broken on 1.8.0 alpha 2

### ISSUE THREAD

**Issue by AlienTux** [Zettlr/Zettlr#1337]
## Description

Exporting doesn't seem to work on 1.8.0 alpha 2. To any format.

## Reproducing

I just have a document and try to export to anything (including HTML5). It always returns the same error:

`The "path" argument must be of type string. Received type undefined`

I do have custom paths to pandoc and xelatex, but AFAIK HTML5 doesn't require a path.

## Expected behaviour

Document should be exported.

## Platform
 - OS and version: Linux, Solus 4.1 Gnome Edition
 - Zettlr Version: 1.8.0 alpha 2

**Comment by nathanlesage** [comment]
Cannot reproduce, for me exporting works fine, both for projects, for single files, with and without custom paths. I never saw that error before. Is the error coming from Pandoc, or from Zettlr?

**Comment by AlienTux** [comment]
Thank you for your reply!

I actually managed to narrow it down a bit. I erased my `/.config/zettlr` folder and the HTML export worked again. I reset my custom path to Pandoc and Xelatex and the exports work except for revealJS (which is a big problem for me).

The error with revealJS is the following: "Cannot read property 'toLowerCase' of undefined". This doesn't happen with the latest stable release.

However, the previous error happens when I changed the export target to the "Current directory". Once I do that no export works.
